QuantConnect.TDAmeritrade/TDAmeritradeBrokerage.WebSocket.cs

```diff
@@ -204,7 +212,7 @@ public void Login()
         {
```

Comment:
`Login()` method should be private, same `LogOut()`, please review the rest of the public methods 👍

Reply:
I changed access modifier in these method and several in RestAPI file.